core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -2524,196 +2528,166 @@ class KafkaApisTest {
     assertEquals(MemoryRecords.EMPTY, FetchResponse.recordsOrFail(partitionData))
   }
 
-  @Test
-  def testJoinGroupProtocolsOrder(): Unit = {
```

Comment:
This test's logic is not tested in `GroupCoordinatorAdaptorTest`.

Reply:
do you mean "now"?

Reply:
Right. Let me fix it.